src/components/families/family-sidebar/family-sidebar-enrolment-card/FamilySidebarEnrolmentCard.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
+
+import { Box, Divider, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  KeyboardArrowDownRounded,
+  KeyboardArrowUpRounded,
+} from "@material-ui/icons";
+
+import StatusChip from "components/common/status-chip/StatusChip";
+import EnrolmentStatus from "constants/EnrolmentStatus";
+import { Student } from "types";
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    backgroundColor: theme.palette.backgroundSecondary.default,
+  },
+  content: {
+    "& .MuiTypography-body2": {
+      color: theme.palette.text.secondary,
+      fontSize: 12,
+      fontWeight: 500,
+    },
+  },
+  session: {
+    "& .MuiTypography-body2": {
+      fontWeight: 700,
+    },
+  },
+  days: {
+    "& .MuiTypography-body2": {
+      fontWeight: 700,
+      color: "#0052CC", // not sure if this colour is used anywhere else in the app, should add to theme if yes (taken from figma)
```

Comment:
ooh good callout, i'm actually not sure if this blue means anything other than just being blue 😅 i imagine it could signify being a link, or if that's a colour-coded class (see #82). could we double check with design?